I wish_If only/Bohdanov_hw.md

```diff
@@ -0,0 +1,41 @@
+# Exercise 1. Please translate the sentences into English
+
+>Жаль что вы не пришли пораньше, чтобы оценить всю ущербность архитектуры предложенной студентом первокурсником.
+
+If only you had came up earlier to (evaluate / estimate) all the lameness of architecture that was proposed by first year student.
+
+>Обидно, что мы не должались его прихода, иначе бы мы были в курсе на каком этапе сейчас находится проект и что нам делать с этим дальше.
+
+If only we had waited for his arrival, otherwise we would be abreast on which stage is the project right now and what we should do with it futher.
+
+> Жаль что мы так и не встретились перед отъездом и не обсудили причину твоего увольнения, у меня много мыслей на этот счет и я хотел бы поделиться ими с тобой.
+
+I wish we had met before departure and discussed a reason of your quit, I had a lot of thoughts on this topic, and I wished to share it with you
+
+>Жаль что проект завершился так и не начавшись ввиду нехватки кадров и соответствующего финансирования. Если бы все сложилось то задумка вполне могла выстрелить на рынке и стать новым хитом.
+
+If only the project has ended up before it started because of lack of personnel and corresponding funding. If everything has happened as it supposed to, plan would succeed and became a new hit. 
+
+>Он пожалел, что пришел и привел с собой друга. На его месте я бы так не поступал. Если бы он пришел вовремя, этого могло и не случиться.
+
+If only he hadn't come and brought a friend with him. I wouldn't do this on his place. If he had come right in time, it might not have happened.
+
+> Если бы не ты я не знаю что бы я тогда ответил заказчику, ты повел разговор совсем в другом направлении и нам удалось избежать серьезного разговора о горящих сроках.
+
+If only you wasn't here I don't know what I would have said to the customer yoi have led the conversation in the another direction, and we succeeded to avoid a serious conversation about pressing deadlines.
+
```

Comment:
`But for you I don't know what I would have answered to the client`
`you led the conversation` - normally we do not duplicate complex constructions but simplify them
the = another - don't duplicate functionality